gravatar-ui/api/gravatar-ui.api

```diff
@@ -0,0 +1,299 @@
+	public synthetic fun <init> (Ljava/lang/Integer;Ljava/lang/Integer;Ljava/lang/Integer;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public final fun component1 ()Ljava/lang/Integer;
+	public final fun component2 ()Ljava/lang/Integer;
+	public final fun component3 ()Ljava/lang/Integer;
+	public final fun copy (Ljava/lang/Integer;Ljava/lang/Integer;Ljava/lang/Integer;)Lcom/gravatar/ui/ImageEditionStyling;
+	public static synthetic fun copy$default (Lcom/gravatar/ui/ImageEditionStyling;Ljava/lang/Integer;Ljava/lang/Integer;Ljava/lang/Integer;ILjava/lang/Object;)Lcom/gravatar/ui/ImageEditionStyling;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getStatusBarColor ()Ljava/lang/Integer;
+	public final fun getToolbarColor ()Ljava/lang/Integer;
+	public final fun getToolbarWidgetColor ()Ljava/lang/Integer;
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public abstract class com/gravatar/ui/components/ComponentState {
+	public static final field $stable I
+}
+
+public final class com/gravatar/ui/components/ComponentState$Empty : com/gravatar/ui/components/ComponentState {
+	public static final field $stable I
+	public static final field INSTANCE Lcom/gravatar/ui/components/ComponentState$Empty;
+	public fun equals (Ljava/lang/Object;)Z
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class com/gravatar/ui/components/ComponentState$Loaded : com/gravatar/ui/components/ComponentState {
+	public static final field $stable I
+	public fun <init> (Ljava/lang/Object;)V
+	public final fun component1 ()Ljava/lang/Object;
+	public final fun copy (Ljava/lang/Object;)Lcom/gravatar/ui/components/ComponentState$Loaded;
+	public static synthetic fun copy$default (Lcom/gravatar/ui/components/ComponentState$Loaded;Ljava/lang/Object;ILjava/lang/Object;)Lcom/gravatar/ui/components/ComponentState$Loaded;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getLoadedValue ()Ljava/lang/Object;
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class com/gravatar/ui/components/ComponentState$Loading : com/gravatar/ui/components/ComponentState {
+	public static final field $stable I
+	public static final field INSTANCE Lcom/gravatar/ui/components/ComponentState$Loading;
+	public fun equals (Ljava/lang/Object;)Z
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class com/gravatar/ui/components/ComponentStateKt {
+	public static final fun LoadingToLoadedStatePreview (Lkotlin/jvm/functions/Function3;Landroidx/compose/runtime/Composer;II)V
+}
+
+public final class com/gravatar/ui/components/ComposableSingletons$ComponentStateKt {
+	public static final field INSTANCE Lcom/gravatar/ui/components/ComposableSingletons$ComponentStateKt;
+	public static field lambda-1 Lkotlin/jvm/functions/Function3;
+	public fun <init> ()V
+	public final fun getLambda-1$gravatar_ui_release ()Lkotlin/jvm/functions/Function3;
```

Comment:
Do you know what's `gravatar_ui_release`? This look suspicious for a public contract.

Reply:
🤔 There's an open issue in the binary compatibility repository: https://github.com/Kotlin/binary-compatibility-validator/issues/74

Reply:
Thanks @hamorillo , TIL!

Reply:
Good catch 👍 Let's keep an eye on that one